github/activity_events_test.go

```diff
@@ -284,8 +284,12 @@ func TestActivityService_EventPayload_typed(t *testing.T) {
 	}
 
 	want := &PushEvent{PushID: Int(1)}
-	if !reflect.DeepEqual(event.Payload(), want) {
-		t.Errorf("Event Payload returned %+v, want %+v", event.Payload(), want)
+	got, err := event.Payload()
+	if err != nil {
+		t.Errorf("Payload = %v, want nil", err)
```

Comment:
This should be `t.Fatalf`, not `t.Errorf`. If `event.Payload()` returned a non-nil error, there's no point in continuing testing the `got` value, it's only noise. /cc @gmlewis

Also in 2 similar places below.